### DISCUSSION TITLE
TypeScript 타깃을 es2020으로 변경?

### DISCUSSION THREAD

**Discussion by saschanaz** [Alex4386/turnoff-namuwiki#62] [General]
현재 target 값이 ES5로 설정되어 있어 async/await 및 for 루프가 들어간 코드를 디버깅할 경우 다소 불편함이 있습니다. 제가 알기로 나무위키 꺼라 확장 기능이 지원하는 모든 브라우저가 최신 표준을 지원하므로 es2020을 사용하면 디버깅이 더욱 용이할 듯한데 어떤가요?

딱히 이슈는 아니므로 토론장에 올립니다.

**Comment by Alex4386** [comment]
@saschanaz 추후 웹팩으로 전환 예정인 관계로 미뤄두었으나, 제가 현재 여기에 몰두할 수 있는 상황이 아닌 관계로 천천히 진행 할 예정입니다.

**Comment by Alex4386** [comment]
webpack 으로의 마이그레이션이 완료되어 Discussion을 닫습니다.
